orchagent/pfcwdorch.cpp

```diff
@@ -480,6 +502,11 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
 
         losslessTc.insert(i);
     }
+    if (losslessTc.empty())
+    {
+        SWSS_LOG_ERROR("No lossless TC found on port %s", port.m_alias.c_str());
+        return false;
```

Comment:
there could be a case where no lossless tc is configured, and then pfc wd should not be started in this case. I am not sure you should return ERROR here.

Reply:
If no lossless tc is configured on a port, what is the point of starting pfc watchdog? It is a misconfig in the first place.

Reply:
not exactly, people might enable pfc watchdog and would expect it is not active until lossless queue is configured on the port.

Reply:
What do you mean by enable pfc watchdog?